src/components/grid/grid.tsx

```diff
@@ -1,26 +1,85 @@
 import styled, { CSSProp } from 'styled-components';
 import { up } from '../breakpoint/breakpoint';
 
+type GridItemSize = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+
 const GRID_GAP_DESKTOP: CSSProp = '24px';
 const GRID_GAP_MOBILE: CSSProp = '12px';
```

Comment:
Where are these values coming from? The notion suggests the value is not changeable, in the design however I see no *one* gap value for desktop and mobile respectively.

Reply:
They are currently not used. I just added it as other grid implementations (like the one from material-ui) also allow to set a grid spacing. If we figure out that we don't need it, we will throw it out.